### DISCUSSION TITLE
<p> tag in <p> tag

### DISCUSSION THREAD

**Discussion by arthur-crahe** [themesberg/flowbite-react#887] [Q&A] [answered]
Hi, I noticed an issue in the documentation related to the following structure:

```
<p className="mb-2 text-gray-500 dark:text-gray-400">
  <p>
    Flowbite is an open-source library of interactive components built on top of Tailwind CSS including buttons,
    dropdowns, modals, navbars, and more.
  </p>
</p>
```
This structure creates an error, which seems to be expected due to the HTML standards: 

![image](https://github.com/themesberg/flowbite-react/assets/83074639/34f90ad6-fd31-4d9b-a57a-8869f1fd6546)

I wanted to ask whether this is an error in the documentation, or if it is intentionally designed this way?

**Comment by tulup-conner** [comment, marked as answer]
Yeah, this is an error with the documentation. We're using Markdown now, and it creates paragraph tags automatically and, in some cases, undesirably. I think this is specifically a problem for Accordion.
